.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: ruby
+
```

Comment:
:information_desk_person: Also recommend adding `cache: bundler` in here as well, which will also speed up tests.